<launch>
    <arg name="device_file" default="/dev/input/mouse1" />
    <node name="mouse_listener" 
          pkg="wouse" 
          type="mouse_listener.py"
          output="screen" 
          args="$(arg device_file)" />
</launch>
